tests/test_match.py

```diff
@@ -117,6 +119,59 @@ def test_that_match_all_obeys_region():
         (177, 75), (177, 119)]]
 
 
+# These ImageLogger tests don't verify the html & png debug output, just that
+# it doesn't raise any exceptions.
```

Comment:
Hmm.  Will putting a `try: except:` block around `_log_match_image_debug` (like we discussed) remove the value from these tests?  Maybe they should at-least check that `index.html` has been written.  You could even use a regex like `<img [^>]*src=['"]([^'"]*)['"] [^>]*>` on index.html to check that all the images you've written are referenced and that all the referenced images were written.